gradle-plugin/configure/build.gradle.kts

```diff
@@ -1,28 +1,28 @@
-import com.novoda.gradle.release.PublishExtension
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 /// The plugin version number – change this to match whatever your tag will be
-version = "0.5.0"
 group = "com.automattic.android"
+version = "0.6.0"
 
 buildscript {
     repositories {
-        jcenter()
-        maven {
-            setUrl("https://plugins.gradle.org/m2/")
-        }
+        maven { url = uri("https://a8c-libs.s3.amazonaws.com/android") }
     }
     dependencies {
-        classpath("com.novoda", "bintray-release", "0.9.2")
+        classpath("com.automattic.android:publish-to-s3:0.3.1")
     }
 }
 
 plugins {
     `kotlin-dsl`
-    id("com.github.gmazzo.buildconfig") version "2.0.2"
+    id("com.github.gmazzo.buildconfig") version "3.0.0"
 }
```

Comment:
For my own knowledge: why do we declare those plugins here in `plugins`, but declare the `publish-to-s3` plugin in `dependencies` and not as a plugin here like the others? 🤔  What's the difference between those?

Reply:
Due to an issue in Gradle where it looks for a jar that's not supposed to be there. More info in the `Lack of Plugin DSL support` section of the [publish-to-s3-gradle-plugin README](https://github.com/Automattic/publish-to-s3-gradle-plugin#lack-of-plugin-dsl-support).